vmdb/app/controllers/miq_policy_controller/miq_actions.rb

```diff
@@ -254,10 +254,7 @@ def action_build_edit_screen
       end
     end
 
-    @edit[:new][:scan_profiles] = Array.new
-    ScanItemSet.all.sort{|a,b| a.name <=> b.name}.each do |sp|
-     @edit[:new][:scan_profiles].push(sp.name)
-    end
+    @edit[:new][:scan_profiles] = ScanItemSet.order(:name).pluck(:name)
```

Comment:
A number of these cases can be reduced in whole, or in part, to normal Arel methods